src/PooledArrays.jl

```diff
@@ -601,4 +601,14 @@ _perm(o::F, z::V) where {F, V} = Base.Order.Perm{F, V}(o, z)
 
 Base.Order.Perm(o::Base.Order.ForwardOrdering, y::PooledArray) = _perm(o, fast_sortable(y))
 
+function Base.repeat(x::PooledArray{T, R, N, RA}, counts...) where {T, R<:Integer, N, RA}
```

Comment:
I would assume that the `{T, R, N, RA}` part is not needed. Why have you added it?

Reply:
based on CI output, it seems it is needed for `Julia 1.0.5`?